### ISSUE TITLE
Plowing problem with stock plough - Plow offset and turned on the wrong side

### ISSUE THREAD

**Issue by MAOGAMI** [Courseplay/courseplay#3535]
Hello, since FS17, now in FS19 I have problem with setting up my plowing course. When I generate a course and tell the CP to start working on it - he's plowing like in the screenshot. He is turning plough wrong sides, the wheel shouldn't be at plowed side but the opposite should be?

Stock JD tractor and stock AgroMasz plough.

![image](https://user-images.githubusercontent.com/28206886/55682818-abfc7380-5938-11e9-8867-3e15e2d8130f.png)

**Comment by Tensuko** [comment]
I don't rly see this is wrong.
Traktor stays outside of the plough area.
Why should it drive on the ploughed side ?

**Comment by Raints** [comment]
@Tensuko  I think MAOGAMI means plough wheel, plough is turned in wrong way.

**Comment by MAOGAMI** [comment]
@Raints  that's what i mean

**Comment by Tensuko** [comment]
You mean this would be the right way ?
![fsScreen_2019_04_07_20_08_20](https://user-images.githubusercontent.com/27344744/55687751-eda91080-5970-11e9-966f-aee3fae5ccbf.png)

**Comment by MAOGAMI** [comment]
yes

**Comment by MAOGAMI** [comment]
nothing new in there?

**Comment by pseikoNL** [comment]
@maogami, as you can see the issue is labelled under ‘enhancement’, so it’s on the todo list of the dev team, but they have more to do as only your issue, so please be patient and wait until the dev team has changed the code for this, you will be notify when done.

Kind regards,
PseikO

**Comment by Tensuko** [comment]
We know about the Issue and will do something about it, but that will come with one smaler fine tuning update.
Just wait a bit more, it stays on our list ;)

**Comment by MAOGAMI** [comment]
Ah sorry then and thanks

**Comment by pvaiko** [comment]
BTW, if you start the course with the plow on the correct side then everything works fine, right?

**Comment by MAOGAMI** [comment]
Worker keeps the plough on the plowed side after turning at the end of the track. When he turns I have then to stop the course, manually turn the plough and start the course again. Any only after that the course is doing fine.

**Comment by pvaiko** [comment]
Ok. Try again. If you set the plow in the correct position *before* you start the course at the very first waypoint it will stay on the correct size on the first row and rotated it into the correct position at the first and all subsequent turns, correct?

**Comment by MAOGAMI** [comment]
I will record something about this case as soon as possible

**Comment by MAOGAMI** [comment]
Alright, there's "documentation" for my problem :d

Link 1 (course generation, found bug with enhancedVehicle mod at the beginning of the course, low speed while driving to 1st point) -> https://youtu.be/jW8irA8nSGU

Link 2 (main description about problem) -> https://youtu.be/Cxnj5C0Am8M

**Comment by Tensuko** [comment]
Try with 268 or higher.

**Comment by biomex81** [comment]
Until version 256 no problem, but when i updated to 268 i noticed some lag spikes and a problem on unload a combine with the usual trailer. Please fix it, thx.

**Comment by Tensuko** [comment]
New Issue with logfile.
And a better description.

**Comment by biomex81** [comment]
When the trailer is full, the tractor can't go to unload because when it reach the start point to the silo route, there are many lags spikes, and the tractor moves around the start point , without being able to start the route to the silo. I hope I explained myself. Thank you so much.

**Comment by pvaiko** [comment]
Closing this, open new issue for any plow related problems.
